lib/autoyast.rb

```diff
@@ -23,6 +23,15 @@ def initialize(description)
       "repositories",
       "packages"
     )
+    if !description.users
+      Machinery::Ui.puts(
+        "\nWarning: Exporting a description without the scope 'users' as AutoYaST" \
+        " profile will result in a root account without a password which prevents" \
+        " logging in.\n" \
+        "So either a section for the root user needs to be added manually to the" \
```

Comment:
I would write this in active form "So either add a section ... or inspect before ..."